Branch-SDK/Branch-SDK/BranchNotifications.h

```diff
@@ -0,0 +1,31 @@
+//
+//  BranchNotifications.h
+//  Branch-TestBed
+//
+//  Created by edward on 6/30/17.
+//  Copyright © 2017 Branch Metrics. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+@class Branch;
+
+@protocol BranchDelegate <NSObject>
```

Comment:
I think the changes you suggested sound good. I'm sure this will evolve if there are more things that are really needed. :+1: